examples/tutorials/01_basics/08_animation.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2016 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Play an animation on Cozmo
+
+Play animations on Cozmo
```

Comment:
Maybe we could advise other animations (triggers and names) they could try and where they can find a list of animations if we don't want to list them here, as the list is lengthy. Would be good if there is a nice list they can look at somewhere and copy and paste from that list.